src/main/java/org/jitsi/jicofo/conference/JitsiMeetConferenceImpl.java

```diff
@@ -2425,6 +2443,12 @@ public void bridgeAdded(Bridge bridge)
 
     private class ChatRoomListenerImpl implements ChatRoomListener
     {
+        private final JitsiMeetConferenceImpl conference;
+        private ChatRoomListenerImpl(JitsiMeetConferenceImpl conference)
```

Comment:
Since `ChatRoomListener` is an inner class of `JitsiMeetConferenceImpl` it has an implicit reference to it. You can just call `numVideoSendersChanged()` (see e.g. how `roomDestroyed` calls `JitsiMeetConferenceImpl.stop()`).

Reply:
thanks.